ui/apps/platform/src/Containers/Risk/Process/ProcessDiscoveryCardHeader.tsx

```diff
@@ -56,19 +54,18 @@ function ProcessDiscoveryCardHeader({
     const trimmedName = name.length > 48 ? `${name.substring(0, 48)}...` : name;
     const style = suspicious
         ? {
-              backgroundColor:
-                  'var(--pf-t--temp--dev--tbd)' /* CODEMODS: original v5 color was --pf-v5-global--palette--red-50 */,
+              border: '1px solid var(--pf-t--global--border--color--status--danger--default)',
```

Comment:
There isn't a real PatternFly pattern to represent a light-red (light danger?) background. The border color change retains the idea here and is a bit more in line with the design of PF6.

Either way, this should be replaced in the upcoming Risk migration.